### PR TITLE
Support uploading screenshots to itunes via `deliver`

### PR DESCRIPTION
**Pull request by orta**
This adds support for downloading/uploading metadata. I've not looked at the IPA bits yet.

### CODE REVIEW

File: lib/deliver/app_screenshot.rb [hunk 1]
```diff
@@ -85,7 +91,12 @@ def self.calculate_screen_size(path)
 
       raise "Could not find or parse file at path '#{path}'" if size.nil? or size.count == 0
 
+      # the iOS TV clashes with the 6+ :-/
+
       devices = {
+        ScreenSize::IOS_TV => [
+          [1920, 1080]
+        ],
```
Comment (orta) [inline lib/deliver/app_screenshot.rb]: so a landscape iPhone 6S resolution _is the same_ as an Apple TV screenshot. I'm not sure this method of keeping track of the platform by the size will work anymore.

Comment (KrauseFx) [inline lib/deliver/app_screenshot.rb]: Uh, that's not good. How about just adding the convention that Apple TV screenshots must contain `TV` in the file name?

**Comment by KrauseFx** [comment]
@orta, not sure why I didn't reply to this back then. This is now implemented in `deliver` and will be possible in the next release :+1:

**Comment by orta** [comment]
even better

**Pull request closed without merging**